docs/telemetry.md

```diff
@@ -4,7 +4,7 @@ Fenix uses Mozilla's telemetry service (Glean) and LeanPlum to measure feature p
 
 ## Baseline ping
 
-Fenix creates and tries to send a "baseline" ping when the app goes to the background. This baseline ping is defined by the [Glean](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/pings.md) component and [documented in the Android Components repository](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/pings/baseline.md).
+Fenix creates and tries to send a "baseline" ping when the app goes to the background. This baseline ping is defined by the [Glean](https://github.com/mozilla/glean/tree/master/docs/user/pings) component and [documented in the Android Components repository](https://github.com/mozilla/glean/blob/master/docs/user/pings/baseline.md).
```

Comment:
Does it make sense to replace the ping descriptions here with a link to [this autogenerated doc](https://github.com/mozilla-mobile/fenix/blob/master/docs/metrics.md)? This doc, in addition to the metrics being sent, also contains ping descriptions.